utils/rpms/cart.spec

```diff
@@ -13,9 +11,11 @@ Source0:       %{name}-%{version}.tar.gz
 Source1:       scons_local-%{version}.tar.gz
 
 BuildRequires: scons >= 2.4
-BuildRequires: libfabric-devel
 BuildRequires: openpa-devel
-BuildRequires: mercury-devel = %{mercury_version}
+BuildRequires: mercury-devel < 2.0.0a1
```

Comment:
It seems like there's a lot of flip-flopping between using the macro version and hard-coding it... Maybe I don't understand the relationship between the PRs.

Reply:
Yeah, because the concept with the macro was that it was an exact version but now we are migrating to _any version less than_.  I don't want people mistakingly continuing to populate the macro as an exact version when it's actually being used as a _less than_ as the results would be not at all what they are looking for.

This really is just a 1.0 branch issue, for future supportability.